src/ec/packages/ec-component-accordion/accordion.html.twig

```diff
@@ -0,0 +1,69 @@
+{#
+  Parameters:
+    - "items" (array) (default: []): format: [
+      {
+        "id": (string),
+        level: (string),
+        toggle: (predefined structure): see Button component, format: {
+          label: (string),
+          icon: (predefined structure): see Icon component,
+        },
+        content: (string),
```

Comment:
I don't know if that would be easy (or even possible), but that would be better to use blocks for accordion content, as it may not be simple text

Reply:
I made the change in a way to allow for a block in the place of the content if the item would contain one. I tried to think of an example, but couldn't reach a valid twig syntax.

I tried to override `item.content` in a loop with a block. I couldn't manage to change items of an iterable from one format to another. Please have a look at the latest change 540b81c and let me know if that was what you had in mind.

If you have a scenario how we can test that in a composition, please let share.